latest_storie_css.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+    <head>
+       <!-- <meta name="viewport" content="width=deive-width, initial-scale=1.0"> -->
+        <style>
+            
+            body{
+                display:block;
+                margin: 1px;
+                
+            }
+            body > div{
+                padding-inline-start: 1px;
+            }
+            .grid-container{
+                display: grid;
```

Comment:
Grid is not required for this simple example.  Grids are mainly used for complex layouts